packages/block-library/src/page-list/editor.scss

```
@@ -4,13 +4,29 @@
 	.wp-block-page-list {
 		background-color: inherit;
 	}
+
 	// Make the dropdown background white if there's no background color set.
 	&:not(.has-background) {
 		.submenu-container {
 			color: $gray-900;
 			background-color: $white;
 		}
 	}
+
+	// space-between justification.
+	&.items-justified-space-between {
+		.wp-block-page-list > div,
+		.wp-block-page-list {
+			flex: 1;
+		}
+
```

I added the following and now it works as I think it was expected to:

```suggestion
		.wp-block-page-list > div,
		.wp-block-page-list {
			display: contents;
			flex: 1;
		}

```

https://user-images.githubusercontent.com/1157901/111011869-9b04da00-8379-11eb-933b-c8d4c456e51a.mov